app/models/job_execution.rb

```diff
@@ -207,6 +207,11 @@ def commands(dir)
       TAG: (@job.tag || @job.commit)
     }.merge(@env)
 
+    deploy = @job.deploy
+    if deploy && deploy.changeset.commit_range.is_a?(String)
```

Comment:
`if deploy = @job.deploy` will be good enough ... I though commit-range is some kind of array :D